v2/cli/term/event_page.go

```diff
@@ -20,6 +20,8 @@ type eventPage struct {
 	eventInfo  *tview.TextView
 	workerInfo *tview.TextView
 	jobsTable  *tview.Table
+	eventLogs  *tview.TextView
```

Comment:
I would suggest changing this to `workerLogs`.

Events are really just "things that happened." Workers are the things that _handle_ the events. So events don't technically have a status. They don't technically have logs. They _do_ each have an associated worker and its worker has a status, logs, etc.